CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing
+
+🥳 Welcome to the party! We are glad you are taking the first step to look at contributing to this community project.
+
+This guide provides an overview of how the project is organized (we have a few different repos) and go over a few of the unique things with our project to help you better understand how you can contribute.
+
+## Project Structure
+
+As of 2023 the project has been restructured but we also have a few different repositories in play to support [dbatools](https://dbatools.io). Most of these you do not have to worry about, but so you are aware of how things are tied together.
+
+1. [dataplat/dbatools](https://github.com/dataplat/dbatools)
+
+    This is the primary project contributions are needed as it hosts all the PowerShell codebase for the 600+ PowerShell commands.
+
+1. [dataplat/dbatools.library](https://github.com/dataplat/dbatools.library)
+
+    As of v2 release the dbatools library, SMO library, and 3rd party libaries have been split out into both a separate repository and [PowerShell module](https://powershellgallery.com/packages/dbatools.library).
+
+1. [dataplat/appveyor-lab](https://github.com/dataplat/appveyor-lab)
+
+    We use Appveyor for running the Pester test for each function. We have this repository to store content that is used in our tests. This keeps the current repository clean from excess files just for testing. **Maintainers will determine if tests require files be placed in this repository**.
+
+1. [dataplat/docs](https://github.com/dataplat/docs)
+
+    This repository and content are auto-generated from the comment-based help (CBH) within each command for the module. Any things found wrong in a given command's help content is addressed in the dbatools module's repository.
+
+1. [dataplat/web](https://github.com/dataplat/web)
+
+    This repository is were the [dbatools.io](https://dbatools.io) site content is hosted.
+
+    > **NOTE** Blog post content **is not** kept updated, it is after all post from scenarios or situations at the time of posting by the author(s).
```

Comment:
Not sure this sentence makes sense. Maybe just changing post to posts, or maybe

```suggestion
    > **NOTE** Blog post content **is not** kept updated, they are descriptions of scenarios or situations at the time of posting by the author(s).
```

Reply:
Decided to just remove.